recipes/metaphlan2/2.9.0/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "metaphlan2" %}
-{% set version = "2.9.20" %}
-{% set md5 = "88e7de3bb3f93f5ea6cd476c3ed09aae" %}
+{% set version = "2.9.21" %}
+{% set md5 = "2caf3765834d121a1be183fbde60dbac" %}
```

Comment:
Can you move this to `recipes/metaphlan2/`? It makes sense to have the newest version there.